docs/modules/user-guide/partials/proc_adding-components-to-a-devfile.adoc

```diff
@@ -12,7 +12,22 @@ Each component in a single devfile must have a unique name and use one of the ob
 |container
 |componentObject
 |no
-|The list of containers that you use.
+|The list of containers.
+
+|kubernetes
+|componentObject
+|no
+|The Kubernetes cluster.
+
+|openshift
```

Comment:
This is a general comment about this page: I'm having a hard time putting the first and second columns together in my head. Is there a way to put them together in one merged table cell, for example as a (one-line or two-line, whatever it looks like in a devfile) code block, so as to see their relationship to each other?

Reply:
I mean something like
`- name: <string>`